tests/unit/core/test_api_errors.py

```
@@ -0,0 +1,661 @@
+        b'{"chunk":"second"}\n',
+    ]
+
+
+@pytest.mark.asyncio
+async def test_declared_bounded_success_json_stops_buffering_after_limit():
+    sent_messages = []
+    prefix = b'{"data":"'
+    oversized_chunk = b"x" * (64 * 1024 + 1)
+    suffix = b'"}'
+
+    async def app(scope, receive, send):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `app`

Add return type annotation: `None`

(ANN202)